apps/web/middleware.ts

```diff
@@ -18,7 +16,33 @@ const safeGet = async <T = any>(key: string): Promise<T | undefined> => {
   }
 };
 
+export const POST_METHODS_ALLOWED_API_ROUTES = ["/api/auth/signup", "/api/trpc"];
+// Some app routes are allowed because "revalidatePath()" is used to revalidate the cache for them
+export const POST_METHODS_ALLOWED_APP_ROUTES = ["/settings/my-account/general"];
```

Comment:
https://github.com/calcom/cal.com/blob/main/apps/web/app/settings/(settings-layout)/my-account/general/page.tsx#L19

<img width="488" alt="Screenshot 2025-02-16 at 6 45 07 AM" src="https://github.com/user-attachments/assets/5c7cd799-8f4a-4054-ac49-57ddcd60d34c" />

Currently, this is the only route that uses `revalidatePath` (POST request to the app route)